test/token/ERC721/ERC721PausableToken.behavior.js

```diff
@@ -0,0 +1,120 @@
+const { assertRevert } = require('../../helpers/assertRevert');
+const { sendTransaction } = require('../../helpers/sendTransaction');
+
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+function shouldBehaveLikeERC721PausableToken (accounts) {
+  const firstTokenId = 1;
+  const owner = accounts[0];
+  const recipient = accounts[1];
+  const operator = accounts[2];
+  const data = '0x42';
+  const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+  const safeTransferFromWithData = function (token, from, to, tokenId, opts) {
+    return sendTransaction(
+      token,
+      'safeTransferFrom',
+      'address,address,uint256,bytes',
+      [from, to, tokenId, data],
+      opts
+    );
+  };
+
+  const shouldBehaveLikeUnpausedToken = function () {
```

Comment:
Since we're only adding the pausable modifiers, couldn't we reuse the standard erc721 behavior and test against it when unpaused (see the `ConditionalEscrow` tests for a related example).